x-pack/plugins/apm/dev_docs/query_debugging_in_development_and_production.md

```diff
@@ -0,0 +1,17 @@
+# Query Debugging
+
+When debugging an issue with the APM UI it can be very helpful to see the exact Elasticsearch queries and responses that was made for a given API request. 
```

Comment:
```suggestion
When debugging an issue with the APM UI it can be very helpful to see the exact Elasticsearch queries and responses that were made for a given API request. 
```

Reply:
Thanks for the feedback. Accidentally merged before committing this 🙈